cmake/catkin_package.cmake

```diff
@@ -374,6 +377,15 @@ function(_catkin_package)
       endif()
     endforeach()
 
+    set(PKG_COMPILE_OPTIONS ${${PROJECT_NAME}_COMPILE_OPTIONS})
+    separate_arguments(PKG_COMPILE_OPTIONS)
+    separate_arguments(PROJECT_DEFINES)
```

Comment:
Why is this necessary? The argument is already received as a list from `cmake_parse_arguments`.